index.d.ts

```diff
@@ -208,6 +264,21 @@ declare namespace Emittery {
 		// `[eventName: EventName]: unknown;`
 	}
 
+	/**
+	The data provided as `eventData` when listening for `Emittery.listenerAdded` or `Emittery.listenerRemoved`.
+	*/
+	interface ListenerChangedData {
+		/**
+		The listener that was added or removed.
+		*/
+		listener: (eventData?: unknown) => void;
+
+		/**
+		The name of the event that was added or removed if `on` or `off` was used instead of `onAny` or `offAny`.
```

Comment:
Re https://github.com/sindresorhus/emittery/pull/49#discussion_r389500443. I don't think it's clear enough.

```suggestion
		The name of the event that was added or removed if `.on()` or `.off()` was used, or `undefined` if `.onAny()` or `.offAny()` was used.
```

Reply:
Fixed!